docs/WindowsBuild.md

```diff
@@ -73,21 +72,12 @@ git clone https://github.com/apple/swift-corelibs-xctest swift-corelibs-xctest
 git clone https://github.com/apple/swift-llbuild llbuild
 git clone https://github.com/apple/swift-tools-support-core swift-tools-support-core
 git clone -c core.autocrlf=input https://github.com/apple/swift-package-manager swiftpm
-git clone https://github.com/compnerd/swift-build swift-build
 ```
 
-## Acquire ICU, SQLite3, curl, libxml2 and zlib
+## Dependencies (ICU, SQLite3, curl, libxml2 and zlib)
 
-```
-C:\Python27\python.exe -m pip install --user msrest azure-devops tabulate
-C:\Python27\python.exe swift-build\utilities\swift-build.py --build-id ICU --latest-artifacts --filter windows-x64 --download
-C:\Python27\python.exe swift-build\utilities\swift-build.py --build-id XML2 --latest-artifacts --filter windows-x64 --download
-C:\Python27\python.exe swift-build\utilities\swift-build.py --build-id CURL --latest-artifacts --filter windows-x64 --download
-C:\Python27\python.exe swift-build\utilities\swift-build.py --build-id zlib --latest-artifacts --filter windows-x64 --download
-C:\Python27\python.exe swift-build\utilities\swift-build.py --build-id SQLite --latest-artifacts --filter windows-x64 --download
-```
-
-Extract the zip files, ignoring the top level directory, into `S:/Library`. The directory structure should resemble:
+The instructions assume that the dependencies are in `S:/Library`. The directory
```

Comment:
The instructions now don't say how to get the dependencies; why not?

Reply:
Simply to close make the repository self-contained.  The intent was to remove the references to my repository and the Azure setup.  The ICU dependency is external.  If you have a good way to handle this, Im happy to adjust the instructions accordingly.